README.md

```diff
@@ -1 +1,149 @@
+
+Navigate to the directory where you want to clone the repository. You can use the cd command followed by the directory path.
+
+Run the following command to clone the repository:
+
+- **git clone https://github.com/AndresEspi/animePage.git**
+
+This command will create a new directory called "animePage" and download the files from the repository into it.
+
+### Usage
+
+Once you've cloned the repository, you can open it in your code editor or integrated development environment (IDE) to make any necessary adaptations.
+
+To run the project, you must follow the instructions provided in the project documentation or README file. Usually, a specific command will be specified to start the project. For example, if the project uses a local web server, you can run the following command:
+
+- **npm start**
+
+Asegúrate de cumplir con todos los requisitos previos y dependencias antes de ejecutar el proyecto.
+
+
+### Run test
+
+Make sure you meet all the prerequisites and dependencies before running the project.
+
+- **npm test**
+
+This command will run the tests defined in the project and display the results in the terminal. Make sure all test dependencies are installed before running this command.  
+
+### Deployment
+
+The deployment of the project will depend on your needs and the implementation platform you want to use. Here are some general steps for deployment:
+
+- Make sure you meet the project-specific requirements and configurations for the production environment.
+- Choose a deployment platform or hosting service that fits your needs (for example, Heroku, AWS, Google Cloud, etc.).
+- Follow the documentation or guides of the selected platform to implement your project. This may involve creating a new app or project, configuring deployment settings, and uploading the project files.
+- Run extensive tests in the production environment to make sure everything works correctly.
+- Once you're happy with the deployment, you can share the deployed project by providing the deployment URL or making it publicly accessible as needed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="AndresEspi"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/AndresEspi)
+- Twitter: [@twitterhandle](https://twitter.com/Andres_Esp1nosa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/andres-espinosa-3bba67271/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AndresEspi/animePage/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you find this project useful, please consider supporting it. Your support means a lot and helps motivate me to continue improving.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] please like the previous reviewer noted, Kindly make sure that your README file contains the section Future Features, including 1-3 potential features to be added and make sure that your MIT License working as expected the link is broken kindly fix it.

Reply:
I would like you to please review the line that you indicated to me since I did not find the error, I remain attentive